### PR TITLE
Add RG3D game engine

### PR DESCRIPTION
**Pull request by saveriomiroddi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1201,6 +1201,9 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [Kiss3d](http://kiss3d.org) — A Keep It Simple, Stupid 3d graphics engine written with Rust
   * [oxidator](https://github.com/Ruddle/oxidator) — A real time strategy game/engine written with Rust and WebGPU
   * [Piston](https://www.piston.rs/) — [![Crates.io](https://img.shields.io/crates/v/piston.svg?style=flat-square)](https://crates.io/crates/piston) [![Crates.io](https://img.shields.io/crates/l/piston.svg)](https://github.com/PistonDevelopers/piston/blob/master/LICENSE)
+  * [RG3D](https://rg3d.rs) — Rust Game engine 3D [![Crates.io](https://img.shields.io/crates/v/rg3d.svg)](https://crates.io/crates/rg3d)
+[![license](https://img.shields.io/crates/l/rg3d.svg)](https://github.com/mrDIMAS/rg3d/blob/master/LICENSE.md)
+[![Crates.io](https://img.shields.io/crates/d/rg3d.svg)](https://crates.io/crates/rg3d)
```
Comment (palfrey) [inline README.md:1206]: This should be one line in line with other items

**Comment by palfrey** [comment]
This has conflicts with current master README. Can you fix them please?

File: README.md (hunk 1 above)
Comment (saveriomiroddi) [reply README.md]: Done!

**Comment by saveriomiroddi** [comment]
> This has conflicts with current master README. Can you fix them please?

Done as well :)

**Comment by saveriomiroddi** [comment]
They've changed the GitHub organization. The old link was still valid but the 301 caused the build to fail. I've updated it.

**Pull request merged**